packages/client/src/offline/reducer.ts

```diff
@@ -395,10 +433,16 @@ function reducer(
         templates: {
           certificates: {
             birth: {
-              definition: certificatesTemplates.birth.svgCode
+              id: certificatesTemplates.birth.id,
+              definition: certificatesTemplates.birth.svgCode,
+              fileName: certificatesTemplates.birth.svgFileName,
+              lastModifiedDate: certificatesTemplates.birth.lastModifiedDate
```

Comment:
I think we can directly use the `certificateTemplates` defined above.

Reply:
Fixed